### DISCUSSION TITLE
X40 Ultra no login prompt

### DISCUSSION THREAD

**Discussion by Mofef** [Hypfer/Valetudo#2139] [Q&A / Support] [answered]
### Are you already using Valetudo?

No, but in the process of starting it

### Your questions

Hi,

not sure if I messed up something, I already asked  a few minutes ago, but when I tried to edit I had a 404....?

I'm tryng to install valetudo using the UART method and can't get a login prompt. Am I missing something? 
* New Dreame X40 Ultra (Is in the list of supported robots)
* it was never online
* UART works
* USB Stick was flashed with `dreame_uart_root.img` using dd
* USB Stick blinks 2-5s on plug-in but no login prompt (or any other reaction)
* I retired with 2 more different usb sticks
* I can attach the boot log if it's relevant

Cheers

**Comment by vivia** [comment, marked as answer]
The X40 supports the Fastboot method, not the UART method. Please re-read the rooting instructions for your robot on valetudo.cloud

**Reply by Mofef** [reply]
Oh my... Thank you so much!!
Ever since I checked general compatibility and bought the robot I never checked the list of supported robots again, but went straigth to Install > Dreame.
And then clearly missed that sentence
> Please check the [supported robots](https://valetudo.cloud/pages/general/supported-robots.html) page to find out which method applies to your model of robot.

Sorry :/
